.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "GNSE_OPENOCD_STLINK_DBG",
+            "type": "cppdbg",
+            "request": "launch",
+            // Update `program` with your application elf path and make sure to update `setupCommands`
+            "program": "${workspaceRoot}/Software/build/debug/app/basic/basic.elf",
```

Comment:
You can have a configuration for each app, right? Or isn't that really maintainable?

Reply:
Yes indeed you can have one configuration per app, but then it would be a repetition of the same exact configs, but with different paths.

I think we can show here one example and developers can opt to replicate.